quickwit/quickwit-cli/src/logger.rs

```diff
@@ -39,7 +40,7 @@ pub fn setup_logging_and_tracing(
     level: Level,
     ansi_colors: bool,
     build_info: &BuildInfo,
-) -> anyhow::Result<()> {
+) -> anyhow::Result<EnvFilterReloadFn> {
     #[cfg(feature = "tokio-console")]
     {
         if std::env::var_os(QW_ENABLE_TOKIO_CONSOLE_ENV_KEY).is_some() {
```

Comment:
there is a type error when that feature is enabled `Result<()>` vs `Result<EnvFilterReloadFn>`